src/pages/booking.jsx

```diff
@@ -1,14 +1,67 @@
 // pages/booking.js
 
-import React from 'react';
+import React, { useState } from 'react';
 import BookingStyles from '../pages/booking.module.css'; // Import the CSS module
-import { styles } from './booking.module.css'; // Import your CSS module
```

Comment:
Did you replace this with BookingStyles? I added fade effects in css for card. Not sure if removing this will effect it, what you think?